acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Also, if for any reason we had to pass this file through a Python interpreter this would let the interpreter know to read it in with "utf-8" encoding.